multiapps-mta/src/main/java/org/cloudfoundry/multiapps/mta/util/InflatorUtil.java

```diff
@@ -0,0 +1,52 @@
+package org.cloudfoundry.multiapps.mta.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.function.ObjIntConsumer;
+import java.util.zip.DataFormatException;
+import java.util.zip.Inflater;
+
+import org.cloudfoundry.multiapps.common.ContentException;
+import org.cloudfoundry.multiapps.common.SLException;
+import org.cloudfoundry.multiapps.mta.Messages;
+
+public class InflatorUtil {
```

Comment:
can this class be unit tested because it has low coverage

Reply:
done